src/components/molecules/card/CardHeader.tsx

```diff
@@ -82,9 +80,9 @@ const CardHeader: FC<IProps> = ({ variant, text, road }) => {
             <RoadNumberImage roadNumber={road} />
           </Box>
           <Box ml={'7%'} display="flex" flex={1} justifyContent="flex-end" alignItems="flex-end" height={30}>
-            <Box className={classes.logosContainer} m={i18n.language === 'he' ? '0 0 0 2.5%' : '0 2.5% 0 0'}>
+              <Box className={classes.logosContainer} mx={1}>
               <Logo src={LamasImage} alt={'Lamas'} height={30} />
-            </Box>
+          </Box>
```

Comment:
why indentation is changed here? 
tab sizes should remain the same.
are you using our prettier config?